static/js/event/page.tsx

```diff
@@ -20,6 +20,45 @@
 
 import React from "react";
 
-export function Page(): JSX.Element {
-  return <div></div>;
+import { Property } from "./types";
+
+interface EventPagePropsType {
+  // Stores information about the particular event node
+  // the event page should render.
+  dcid: string;
+  name: string;
+  properties: Array<Property>;
+}
+
+/**
+ * Main component for rendering an event page.
+ * Displays the properties and property values of the event described.
+ */
+export function EventPage(props: EventPagePropsType): JSX.Element {
+  return (
+    <div>
+      <h1>{props.name}</h1>
+      <h3>dcid: {props.dcid}</h3>
+      <br />
+      <h3>Properties:</h3>
+      {props.properties.map((property) => {
+        return (
+          <div key={property.dcid}>
+            <b>{property.dcid}:</b>
+            <ul>
+              {property.values.map((value) => {
+                return (
+                  <li key={JSON.stringify(value)}>
```

Comment:
nit: for the key, maybe we can use: {\`${property.dcid}-${value.dcid}\`} so that uniqueness is guaranteed?

Reply:
Done!